PWGLF/DataModel/LFResonanceTables.h

```diff
@@ -0,0 +1,149 @@
+{
+/// Distinuishes the different daughter types
+enum DaughterType {
+  kTrack,         //! Track
+  kV0,            //! V0
+  kCascade,       //! Cascade
+  kNDaughterTypes //! Number of Daughter types
+};
+static constexpr std::string_view DaughterTypeName[kNDaughterTypes] = {"Tracks", "V0", "Cascade"}; //! Naming of the different particle types
+
+enum PDGtype {
+  kPion = BIT(0),
+  kKaon = BIT(1),
+  kProton = BIT(2),
+  kHasTOF = BIT(6) // Save hasTOF info for TOF selection
+};
+
+#define requireTPCPIDCutInFilter(mask) ((aod::resodaughter::tpcPIDselectionFlag & (uint8_t)aod::resodaughter::mask) == (uint8_t)aod::resodaughter::mask)
+#define requireTOFPIDCutInFilter(mask) (((aod::resodaughter::tofPIDselectionFlag & (uint8_t)aod::resodaughter::kHasTOF) != (uint8_t)aod::resodaughter::kHasTOF) || (((aod::resodaughter::tofPIDselectionFlag & (uint8_t)aod::resodaughter::mask) == (uint8_t)aod::resodaughter::mask) && ((aod::resodaughter::tofPIDselectionFlag & (uint8_t)aod::resodaughter::kHasTOF) == (uint8_t)aod::resodaughter::kHasTOF)))
+#define requireTPCPIDPionCutInFilter() requireTPCPIDCutInFilter(PDGtype::kPion)
+#define requireTPCPIDKaonCutInFilter() requireTPCPIDCutInFilter(PDGtype::kKaon)
+#define requireTPCPIDProtonCutInFilter() requireTPCPIDCutInFilter(PDGtype::kProton)
+#define requireTOFPIDPionCutInFilter() requireTOFPIDCutInFilter(PDGtype::kPion)
+#define requireTOFPIDKaonCutInFilter() requireTOFPIDCutInFilter(PDGtype::kKaon)
+#define requireTOFPIDProtonCutInFilter() requireTOFPIDCutInFilter(PDGtype::kProton)
+
+DECLARE_SOA_INDEX_COLUMN(ResoCollision, resoCollision);
+DECLARE_SOA_COLUMN(Pt, pt, float);                                     //! p_T (GeV/c)
+DECLARE_SOA_COLUMN(Px, px, float);                                     //! p_x (GeV/c)
+DECLARE_SOA_COLUMN(Py, py, float);                                     //! p_y (GeV/c)
+DECLARE_SOA_COLUMN(Pz, pz, float);                                     //! p_z (GeV/c)
+DECLARE_SOA_COLUMN(Eta, eta, float);                                   //! Eta
+DECLARE_SOA_COLUMN(Phi, phi, float);                                   //! Phi
+DECLARE_SOA_COLUMN(PartType, partType, uint8_t);                       //! Type of the particle, according to resodaughter::ParticleType
+DECLARE_SOA_COLUMN(TempFitVar, tempFitVar, float);                     //! Observable for the template fitting (Track: DCA_xy, V0: CPA)
+DECLARE_SOA_COLUMN(Indices, indices, int[2]);                          //! Field for the track indices to remove auto-correlations
+DECLARE_SOA_COLUMN(Sign, sign, int8_t);                                //! Sign of the track charge
+DECLARE_SOA_COLUMN(TPCNClsCrossedRows, tpcNClsCrossedRows, uint8_t);   //! Number of TPC crossed rows
+DECLARE_SOA_COLUMN(DcaXY, dcaXY, float);                               //! DCA_xy
+DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);                                 //! DCA_z
+DECLARE_SOA_COLUMN(X, x, float);                                       //! x position of the track
```

Comment:
This is a redefinition of https://github.com/AliceO2Group/AliceO2/blob/dev/Framework/Core/include/Framework/AnalysisDataModel.h#L98 you could consider avoid this duplication and use directly `aod::X` in your table

Reply:
That's a reasonable suggestion. I'll update the table.